packages/lab/src/checkbox/Checkbox.css

```diff
@@ -41,11 +39,6 @@
 }
 
```

Comment:
to add:

.uitkCheckbox-disabled .uitkCheckbox-label,
.uitkCheckbox-disabled .uitkCheckbox-labelRight {
  --uitkControlLabel-color: var(--uitk-text-primary-foreground-disabled);
}